axelor-sale/src/main/java/com/axelor/apps/sale/service/saleorder/SaleOrderSplitServiceImpl.java

```diff
@@ -146,4 +146,26 @@ public BigDecimal getQtyToOrderLeft(SaleOrderLine saleOrderLine) {
 
   protected void checkBeforeConfirm(SaleOrder saleOrder, Map<Long, BigDecimal> qtyToOrderMap)
       throws AxelorException {}
+
+  @Override
+  public BigDecimal computeCurrentlyTotalOrdered(
+      SaleOrder saleOrder, List<Map<String, Object>> saleOrderLineListContext) {
+    BigDecimal currentlyTotalOrdered = BigDecimal.ZERO;
+    if (CollectionUtils.isEmpty(saleOrderLineListContext)) {
+      return currentlyTotalOrdered;
+    }
+    for (Map<String, Object> lineContext : saleOrderLineListContext) {
+      if (lineContext.get("qtyToOrder") == null) {
+        continue;
+      }
+      SaleOrderLine saleOrderLine =
+          saleOrderLineRepository.find(Long.parseLong(lineContext.get("id").toString()));
+      BigDecimal price =
+          saleOrder.getInAti() ? saleOrderLine.getInTaxPrice() : saleOrderLine.getPrice();
+      BigDecimal qtyToOrder =
+          BigDecimal.valueOf(Double.parseDouble(lineContext.get("qtyToOrder").toString()));
+      currentlyTotalOrdered = currentlyTotalOrdered.add(qtyToOrder.multiply(price));
+    }
+    return currentlyTotalOrdered;
+  }
```

Comment:
please move it to another service, or maybe even keep the code in the controller since it is related to parsing information from the view.